docs/can-observe.md

```diff
@@ -25,33 +26,37 @@ dog.name = 'Wilbur';
 
 ## Use
 
-Using __can-observe__ allows you to create observable objects for which any property added can be observed. It is like [can-map] but without the required [can-map.prototype.attr attr] method. This makes can-observe ideal for use-cases where the data that needs to be observed might be dynamic, or where the more rigid approach of [can-define] is unneeded.
+Using `can-observe` allows you to create observable objects with any property added can be observed, including nested objects. It is like [can-map] but without the required [can-map.prototype.attr attr] method. This makes `can-observe` ideal for use-cases where the data may be dynamic, or where the more rigid approach of [can-define] is unneeded.
 
-To use can-observe call the observe method with an object for which to create a proxy for. This will return a [Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) object for which any changes will then reflect back on the target object.
+To use `can-observe` call the `observe()` method with an object. This will return a [Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) object where any changes will reflect back on the target object. Nested objects will be observed lazily when they are accessed or set dynamically after initialization on a `can-observe` proxy object.
+
+To listen for changes on a property, use [can-reflect/observe.onKeyValue canReflect.onKeyValue]. Pass the handler as the third argument which will be triggered when a new value is set on the proxy. In the example below, updating `dog.name` to `"Wilbur"` will trigger the callback that will `console.log` the new value `"Wilbur"`.
 
 ```js
+var canReflect = require("can-reflect");
 var dog = observe({});
 
-dog.on("name", function(){
-	// changed!
+canReflect.onKeyValue(dog, 'name', function(newVal){
+	console.log(newVal); //-> "Wilbur"
 });
 
 dog.name = "Wilbur";
 ```
 
-can-observe can be combined with any other CanJS observable type, like [can-define] or [can-compute]. In this example we create a compute that changes when a can-observe proxy changes.
+`can-observe` can be combined with any other CanJS observable type, like [can-define] or [can-compute]. In this example we create a compute that changes when a can-observe proxy changes. Note that with computes we use [can-reflect/observe.onValue canReflect.onValue] to set up the event listener and handler.
 
 ```js
 var compute = require("can-compute");
 var observe = require("can-observe");
+var canReflect = require("can-reflect");
 
 var person = observe({})
 
 var fullName = compute(function(){
 	return person.first + " " + person.last;
 });
 
-fullName.on("change", function(){
+canReflect.onValue(fullName, "change", function(){
```

Comment:
use `on`, not `can-reflect`.  `can-reflect` is really a  tool for implementors, not users.